clientlibs/js/package.json

```diff
@@ -7,7 +7,7 @@
   ],
   "scripts": {
     "build:bundler": "webpack",
-    "build:types": "./node_modules/.bin/dts-bundle-generator -o dist/browser/index.d.ts src/index.ts && ./node_modules/.bin/dts-bundle-generator -o dist/node/index.d.ts src/index.ts",
+    "build:types": "./node_modules/.bin/dts-bundle-generator -o dist/browser/index.d.ts src/index.ts && ./node_modules/.bin/dts-bundle-generator -o dist/node/index.d.ts src/index.ts && ./node_modules/.bin/dts-bundle-generator -o dist/browser-lite/index.d.ts src/index.ts && ./node_modules/.bin/dts-bundle-generator -o dist/node-lite/index.d.ts src/index.ts",
```

Comment:
You should add multiple scripts for build here. You can go with nps or something simple like concurrently to manage them.

Reply:
yeah it's out of hand, i can break it into parts

Reply:
updated